src/ConfigParser.cpp

```diff
@@ -1,12 +1,19 @@
 #include "../include/ConfigParser.hpp"
 #include "../include/Server.hpp"
 
-void ConfigParser::clearMap(vectorMap& vMap)
+ConfigParser::ConfigParser() : lineNumber(1)
+{
+}
+
+ConfigParser::~ConfigParser()
 {
-	for (vectorMap::iterator it = vMap.begin(); it != vMap.end(); ++it) {
-		it->second.clear();
-	}
-	vMap.clear();
+}
+
+static void configError(const std::string& str, size_t lineNumber)
+{
+	std::string base = "Config File Error: ";
+	base += str + " Line: " + std::to_string(lineNumber);
```

Comment:
I think itll be safer to use the string function append, concatination is usually quite expensive and overloadable. Just a suggestion, though I've been changing these in other places too!